README.md

```diff
@@ -1 +1,122 @@
-# OME Files Benchmarking scripts
+# OME Files Benchmark
+
+## Introduction
+
+The current repository includes a set of benchmarks used to test the following
+software:
+
+-   [OME Files 0.3.1](http://downloads.openmicroscopy.org/ome-files-cpp/0.3.1/)
+-   [Bio-Formats 5.3.4](http://downloads.openmicroscopy.org/bio-formats/5.3.4/)
+    both using the Java Virtual Machine and the
+    [Jace C++ bindings](https://github.com/ome/bio-formats-jace)
+
+The benchmark suite has been executed under Windows Server 2008 and Ubuntu
+16.04 with the exception of the JACE C++ bindings which we have only built
+under Ubuntu 16.04 using JDK 7.
+
+## Benchmark datasets
+
+The reference datasets used for the benchmark are compose of three public OME-TIFF filesets representing different aspects of the OME Data model:
+
+-   a typical 5D fluorescence image - see  http://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/tubhiswt-4D/
+-   a plate generated from the public [Broad Bioimage Benchmark Collection](https://data.broadinstitute.org/bbbc/) and exported as an OME-TIFF - see http://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/BBBC/ [BBBC](http://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/BBBC/)
+-   a metadata-rich (13K Regions of Interest) time-lapse sequence from the  [Mitocheck](http://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/mitocheck/) project - see http://downloads.openmicroscopy.org/images/OME-TIFF/2016-06/mitocheck/
+
+For more references and description, see https://www.openmicroscopy.org/site/support/ome-model/ome-tiff/data.html.
+
+## Benchmark tests
+
+For each of the datasets above, four benchmark tests were executed:
+
+-   metadata.read: the metadata is extracted from the OME-TIFF tag and
+    converted into the OME Data Model
+-   metadata.write: the metadata is written to disk as an OME-XML file
+-   pixeldata.read: the pixeldata is read from the OME-TIFF and stored in
+    memory
+-   pixeldata.write: the pixeldata is written to disk as another OME-TIFF
+
+For each test, the execution time is measured in the benchmark script
+using the standard system utility function and stored in a tabular form.
```

Comment:
What's the "standard system utility"?  We measure in the program rather than the script.

I would say that the benchmark records the real time in milliseconds before and after each test, and computes the elapsed time from the difference.